src/MongoClient.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace MongoDB;
+
+use MongoDB\Driver\Manager;
+use MongoDB\Database;
+use MongoDB\Collection;
+
+class MongoClient
+{
+    protected $manager;
+    protected $wc;
+    protected $rp;
+
+
+    /**
+     * Constructs new MongoClient instance
+     *
+     * This is the suggested main entry point using phongo.
+     * It acts as a bridge to access individual databases and collection tools
+     * which are provided in this namespace.
+     *
+     * @param Manager        $uri            The MongoDB URI to connect to
+     * @param WriteConcern   $options        URI Options
+     * @param ReadPreference $driverOptions  Driver specific options
+     */
+    public function __construct($uri, $options, $driverOptions)
+    {
+        $this->manager = new Manager($uri, $options, $driverOptions);
+    }
+
+    public function selectDatabase($databaseName, WriteConcern $writeConcern = null, ReadPreference $readPreferences = null)
+    {
+        return new Database($this->manager, "{$databaseName}", $writeConcern, $readPreferences);
```

Comment:
Oh, its because I first did selectCollection() and then selectDatabase() copied from that :)
Fixed